.env

```diff
@@ -7,7 +7,9 @@
 #VITE_BROWSER_ROUTER = ""
 #VITE_REDIRECT_URI = "http://localhost:4444/"
 #VITE_AUTHO_SERVER_ENDPOINT = "ourAutho" 
+#VITE_AUTHO_SERVER_ENDPOINT_MOB = "mobAutho" 
 #VITE_GH_OAUTH_CLIENT_ID = "Ov23liN8Q3iGPXSUHUsH"  
+#VITE_GH_OAUTH_CLIENT_ID_MOB = "Ov23lioNfq4Q063COhYR"
```

Comment:
I think that for security we shouldn't really be publishing these on github, we should be setting them as environmental variables on our systems